raiden/tests/unit/transfer/mediated_transfer/test_targetstate.py

```diff
@@ -712,7 +592,7 @@ def test_target_receive_lock_expired():
     iteration = target.state_transition(
         init_transition.new_state,
         lock_expired_state_change,
-        from_channel,
+        channels[0],
```

Comment:
nit: kwargs here and in all the `target.state_transition()` calls in this file so that I stop repeating myself :stuck_out_tongue: